components/org.wso2.carbon.identity.api.server.userstore/org.wso2.carbon.identity.api.server.userstore.v1/src/main/java/org/wso2/carbon/identity/api/server/userstore/v1/core/ServerUserStoreService.java

```diff
@@ -166,6 +166,7 @@ public UserStoreResponse editUserStore(String domainId, UserStoreReq userStoreRe
         try {
             classNames = userStoreConfigService.getAvailableUserStoreClasses();
             List<AvailableUserStoreClassesRes> propertiesToAdd = new ArrayList<>();
+            Map<String, Boolean> userStoreManagersType = UserStoreManagerRegistry.getUserStoreManagersType();
```

Comment:
Any specific reason to use the `Boolean` object type here? Can't we use primitive (`boolean`)?

Reply:
We cannot use primitive types to declare Map objects. Use of Map<String, boolean> map is not allowed in java.